src/hex_api_organization.erl

```diff
@@ -0,0 +1,18 @@
+-module(hex_api_organization).
+-export([
+    get/1,
+    list/1,
+    update/2
+]).
+
+list(Config) when is_map(Config) ->
+    hex_api:get(Config, ["orgs"]).
+
+get(Config) when is_map(Config) ->
+    Path = hex_api:build_organization_path(Config, []),
+    hex_api:get(Config, Path).
+
+update(Config, Seats) when is_integer(Seats) and is_map(Config) ->
```

Comment:
should we change `Seats` to a map so that we can extend it in the future?

FWIW existing APIs are e.g. `hex_api_key:add(Config, Name, Permissions)` so they're not growable, which I regret. I'm ok with moving forward with this as is, if accepting maps is preferred maybe we should do a full revamp at some point.

Reply:
I am merging this as-is since it is consistent with existing APIs but we should probably move to a maps based API. @wojtekmach can you work on this or open an issue with your ideas/plans?

Reply:
Sounds good, will do.